plugins/modules/mso_schema_site_anp_epg_staticport.py

```diff
@@ -185,6 +215,8 @@ def main():
         type=dict(type='str', default='port', choices=['port']),
         pod=dict(type='str'),  # This parameter is not required for querying all objects
         leaf=dict(type='str'),  # This parameter is not required for querying all objects
+        fex_port=dict(type=bool, default=False),   # This parameter is not required for querying all objects
```

Comment:
Change to type='bool' so that it passes sanity testing.

Reply:
Do we need the fex_port attribute?
We should be able to use the fex attribute for that condition.

Reply:
I believe we could get away with just using the fex attribute if it wasn't being used to specify that fex attribute is required.
`['fex_port', 'true', ['type', 'pod', 'leaf', 'fex', 'path', 'vlan']]`

If we switched to just the fex attribute, we'd have to find a way to require fex if the value of fex was not none/null/empty. I tried a couple of different ways to do that, but it seems like `required_if` is always looking for a key, value and will not take anything other than a raw string as the value. 
Ex. `['fex', is not none, ['type', 'pod', 'leaf', 'fex', 'path', 'vlan']]` This produces a syntax error

Reply:
But you do not need to do that check anymore because all the other parameters are already required for absent or present right?
If 'fex' exists, we build the path from line 255.

Reply:
I see what you are saying. I had not thought about it like that, but you are correct and I was able to completely remove fex_port param.